### PR TITLE
[813] Membership calculator issues

### PR DESCRIPTION
**Pull request by rotator**
More info: #813 issue.

### CODE REVIEW

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build403 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build403 |
| Installation process | http://install.openy-dev.ffwua.com/build403/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/403/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build403/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build403/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build403/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build403/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build403/blogWCAG2AAhtmlcs.txt

File: modules/custom/openy_calc/src/CalcDataWrapper.php [hunk 1]
```diff
@@ -282,25 +282,22 @@ public function getBranchPins() {
...
-          break;
+      if (!$coordinates) {
+        continue;
       }
+      $uri = !empty($typeIcons[$location->bundle()]) ? $typeIcons[$location->bundle()] :
+        drupal_get_path('module', 'location_finder') . "/img/map_icon_green.png";
       $pins[] = [
-        'icon' => $icon,
-        'tags' => $tags,
+        'icon' => file_create_url($uri),
```
Comment (podarok) [inline modules/custom/openy_calc/src/CalcDataWrapper.php]: file_create_url creates absolute url, which is not always good
Can we return only relative path from parse_url here?

**Review by podarok — CHANGES_REQUESTED** [review]
small fix, all other looks good

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build428 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build428 |
| Installation process | http://install.openy-dev.ffwua.com/build428/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/428/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build428/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build428/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build428/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build428/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build428/blogWCAG2AAhtmlcs.txt

**Pull request merged**